### DISCUSSION TITLE
running NequIPCalculator with ASE

### DISCUSSION THREAD

**Discussion by nisargjoshi10** [mir-group/nequip#186] [Q&A] [answered]
I'm trying to run an MD simulation with the trained model using the following code, but cant make it to work. Not sure if I'm using it wrong but if you have any feedback that'd be great:

```
calc_neq = NequIPCalculator(model='water-aimd-vasp.pth', r_max=4.5, device='cpu')
water = read('/Users/nisarg/Desktop/deepmd-kit/packmol/examples/water_52.pdb')
water.set_calculator(calc_neq)
md = NVTBerendsen(water, 0.5 * units.fs, 300, 0.005)
md.run(4000)
```
and the error is `TypeError: 'str' object is not callable`

```
File ~/opt/anaconda3/envs/nequip/lib/python3.8/site-packages/ase/md/md.py:137, in MolecularDynamics.run(self, steps)
    135 """ Call Dynamics.run and adjust max_steps """
    136 self.max_steps = steps + self.nsteps
--> 137 return Dynamics.run(self)
```

I'm new to ASE and I might have messed up the steps for using the calculator as I'm getting the same error for `water.get_potential_energy()`

Thanks for the help!

**Comment by Linux-cpp-lisp** [comment, marked as answer]
You need `NequIPCalculator.from_deployed_model()`, not `NequIPCalculator()`.

**Reply by nisargjoshi10** [reply]
that worked! thanks!
